src/website/src/pages/ec/components/button/docs/usage.mdx

```diff
@@ -9,33 +9,54 @@ import { Paragraph } from '@ecl/website-components';
 
-Each page can have one to two Primary (or Call-to-action) buttons. Any remaining calls-to-action should be represented as Secondary or Ghost buttons.
+| Button name                 | Visual style                                                                                            | Usage                                                                                                                                                                                               |
+| --------------------------- | ------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Call-to-action (CTA) button | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_CTA_Default.png)       | funnel users into the most important action for your business ex: "Apply for funding" or "Watch live streaming". Limit to one CTA on a given page, the same CTA can be repeated within in the page. |
+| Primary button              | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Primary_Default.png)   | for primary actions in forms, ex: "Log in", "Get funding", "Register", "Submit a request", should appear only few occurence on a page, ex: only 1 primary button per form                           |
+| Secondary button            | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Secondary_Default.png) | secondary action that users can perform on the page and it can be used multiple times per page.                                                                                                     |
+| Ghost button                | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Ghost_Default.png)     | when an action does not require dominance on a page, ex: "Cancel" a form.                                                                                                                           |
+| Search button               | ![Button Image](https://inno-ecl.s3.amazonaws.com/media/images/EC/Buttons/Button_Search_Default.png)    | use exclusively for the search function.                                                                                                                                                            |
 
 ### Button labels
 
 Button labels tell users what will happen when they click the button. Use verbs that describe the action, such as Send, Add or Delete. Do not use capitalization on buttons (only the first word in a phrase and proper nouns are capitalized) and no more than three words for button labels.
 
 ### Button icons
 
-It is possible to add an icon to a button to enhance the action.
+It is possible to add an icon to a button to better illustrate the action.
```

Comment:
we could add that the icon may be displayed before or after the label

Reply:
then we should explain what's the difference or at least provide guidelines on when,why to use icons left or right. I don't now any difference so far.